questions.js

```diff
@@ -1 +1,54 @@
 //Your quiz questions array of objects goes here
+document.addEventListener("load", function(event) {
+    //getQuestions();
+});
+
+
+let questions = []
+//PLEASE REMEMBER TO CHANGE THIS TO 0 AT ONE POINT
+//let numOfQuestions = 0;
+
+//Place the item inside of the storage, turning it into JSON
+//sessionStorage.setItem(`question1`, JSON.stringify(questions));
+
+//Get the item from the storage, and parse it out of the JSON
+//console.log(JSON.parse(sessionStorage.getItem(`questions`)));
+
+function makeNewQuestion(ques, ans1, ans2, ans3, ans4, correct)
+{
+    //numOfQuestions += 1;
+    //Right here is where reading from a form is gonna go
+
+    //Right here is where creating the question and pushing it to the questions list is going to go
+    questions = JSON.parse(sessionStorage.getItem(`questions`));
```

Comment:
you should have something to make sure that questions stays as an array if there isn't currently anything in session storage.  It currently causes an error if nothing is in session storage.  you could do something like:
`questions = JSON.parse(sessionStorage.getItem(`questions`))??[];`